build.py

```diff
@@ -10,7 +14,7 @@ def wait_until_container_is_ready():
     matched_line=""
     while not done:
         time.sleep(5)
-        result=subprocess.run(['oc','-n','acct-mgt','get','all'],stdout=subprocess.PIPE,stderr=subprocess.STDOUT) 
+        result=subprocess.run(['oc','-n','acct-mgt-2','get','all'],stdout=subprocess.PIPE,stderr=subprocess.STDOUT)
```

Comment:
You can simplify the code a bit by using `subprocess.check_output` instead of `subprocess.run`:

```
out = subprocess.check_output([['oc','-n','acct-mgt-2','get','all'], stderr=subprocess.STDOUT)
lines = out.decode('utf-8').splitlines()
```

`check_output` will raise an exception if the called commands fails. In general consider replacing your calls to `subprocess.run` throughout this code with `subprocess.check_output` (if you need the output) and `subprocess.check_call` (if you don't), because currently you're never checking the return value from your commands.

You can save yourself a call to `decode` by setting `text=True` when calling `subprocess` methods:

```
out = subprocess.check_output([['oc','-n','acct-mgt-2','get','all'], stderr=subprocess.STDOUT, text=True)
lines = out.splitlines()
```

This will decode the bytes using the system encoding, which in almost all cases should be `utf-8`, but if you want to be explicit:

```
out = subprocess.check_output([['oc','-n','acct-mgt-2','get','all'], stderr=subprocess.STDOUT, text=True, encoding='utf-8')
lines = out.splitlines()
```